src/pages/landing/landingController.js

```diff
@@ -1,23 +1,22 @@
 import React from 'react';
-import { useQuery } from '@apollo/client';
-import CircularProgress from '@material-ui/core/CircularProgress';
+// import { useQuery } from '@apollo/client';
+// import CircularProgress from '@material-ui/core/CircularProgress';
 import LandingView from './landingView';
-import { Typography } from '../../components/Wrappers/Wrappers';
-import { GET_LANDING_PAGE_DATA_QUERY } from '../../bento/landingPageData';
+// import { Typography } from '../../components/Wrappers/Wrappers';
+// import { GET_LANDING_PAGE_DATA_QUERY } from '../../bento/landingPageData';
 
-const landingController = () => {
-  const { loading, error, data } = useQuery(GET_LANDING_PAGE_DATA_QUERY);
+const landingController = () => <LandingView />;
+// const { loading, error, data } = useQuery(GET_LANDING_PAGE_DATA_QUERY);
 
```

Comment:
@ogwuazors Can you remove commented code if it's not required?

Reply:
should be resolved now.